cytotable/utils.py

```diff
@@ -453,3 +399,51 @@ def _expand_path(
         modifed_path = modifed_path.expanduser()
 
     return modifed_path.resolve()
+
+
+def _get_cytotable_version() -> str:
+    """
+    Seeks the current version of CytoTable using either pkg_resources
+    or dunamai to determine the current version being used.
+
+    Returns:
+        str
+            A string representing the version of CytoTable currently being used.
+    """
+
+    import dunamai
+
+    import cytotable
+
+    try:
+        # attempt to gather the development version from dunamai
+        # for scenarios where cytotable from source is used.
+        return dunamai.Version.from_any_vcs().serialize()
+    except RuntimeError:
+        # else grab a static version from __init__.py
+        # for scenarios where the built/packaged cytotable is used.
+        return cytotable.__version__
+
+
+def _write_parquet_table_with_metadata(table: pa.Table, **kwargs) -> None:
+    """
+    Adds metadata to parquet output from CytoTable.
```

Comment:
can you add some detail about what some kwargs might represent? Maybe just pointing to the kwargs options in parquet.write_table?

Reply:
Thanks - definitely! I've added an additional **kwargs docstring description to help clarify here.